src/Microsoft.Extensions.Logging.Log4Net.AspNetCore/Microsoft.Extensions.Logging.Log4Net.AspNetCore.csproj

```diff
@@ -30,7 +30,8 @@ https://github.com/huorswords/Microsoft.Extensions.Logging.Log4Net.AspNetCore/is
 
   <ItemGroup>
     <PackageReference Include="log4net" Version="2.0.8" />
-    <PackageReference Include="Microsoft.Extensions.Logging" Version="1.1.2" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="1.1.2" Condition="'$(TargetFramework)' == 'netcoreapp1.1'" />
+    <PackageReference Include="Microsoft.Extensions.Logging" Version="2.0.1" Condition="'$(TargetFramework)' != 'netcoreapp1.1'" />
```

Comment:
Same here. Maybe it is better to create the dependency condition on netcoreapp2.0 instead of assuming netcoreapp1.1 as target version reference. What do you think?

Reply:
Again, when targeting NETCOREAPP2.0, NET462 and NETSTANDARD2.0 then we can use the updated logging library to support ILoggingBuilder.
I just wanted to minimize the writing effort when using netcoreapp1.1 as "reference". Because this is the only target framework which not supports NETStandard2.0 (which Microsoft.Extensions.Logging v2.0 depends on).